data/context/standard_test.go

```diff
@@ -152,6 +171,16 @@ var _ = Describe("Standard", func() {
 				Expect(child).ToNot(BeNil())
 			})
 
+			It("Logger returns a logger", func() {
+				Expect(child.Logger()).ToNot(BeNil())
```

Comment:
i guess your only testing that it is not nil, and not that it is actually a logger ??